packages/deployments/contracts/contracts_forge/messaging/connectors/admin/AdminHubConnector.t.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.17;
+
+import {MockSpokeConnector} from "../../../utils/Mock.sol";
+import {SpokeConnector} from "../../../../contracts/messaging/connectors/SpokeConnector.sol";
+import {AdminHubConnector} from "../../../../contracts/messaging/connectors/admin/AdminHubConnector.sol";
+import {RootManager} from "../../../../contracts/messaging/RootManager.sol";
+import {WatcherManager} from "../../../../contracts/messaging/WatcherManager.sol";
+import {MerkleTreeManager} from "../../../../contracts/messaging/MerkleTreeManager.sol";
+import {Message} from "../../../../contracts/messaging/libraries/Message.sol";
+import {RateLimited} from "../../../../contracts/messaging/libraries/RateLimited.sol";
+
+import "../../../utils/ForgeHelper.sol";
+
+contract AdminHubConnectorTest is ForgeHelper {
```

Comment:
if this is a fork test, you should put it in the `upgrade` directory

Reply:
or otherwise somehow omit from standard unit testing

Reply:
Not sure why, this should be able to be run with normal tests.